public/js/controllers/charts_controller.js

```diff
@@ -336,13 +384,19 @@ export default class extends Controller {
       return node
     }
 
-    this.settings = TurboQuery.nullTemplate(['chart', 'zoom', 'scale', 'bin', 'axis'])
+    this.settings = TurboQuery.nullTemplate(['chart', 'zoom', 'scale', 'bin', 'axis', 'visibility'])
     this.query.update(this.settings)
     this.settings.chart = this.settings.chart || 'ticket-price'
     this.zoomCallback = this._zoomCallback.bind(this)
     this.drawCallback = this._drawCallback.bind(this)
     this.limits = null
     this.lastZoom = null
+    this.visibility = []
+    if (this.settings.visibility) {
+      this.settings.visibility.split(',', -1).forEach(s => {
```

Comment:
Actually, I do not care for how the URL ends up with `%2C` in it, but I don't have a better suggestion for this.  Since the charts and controls are working, I don't want to delay this PR any longer.  For a future issues.  @buck54321 might also have a suggestion for this.

Reply:
The visibility here is an array.
Some chart has multiple lines and checkboxes for toggling the visibility of any of the lines. The encoded value of the array is what lead to the `%2C` from `visibility = [true, false]`.
During the iterations of the PR, I had a recommendation to preserve the visibility across page reload

Reply:
I understand the reason,  I'm just pointing out that in the URL, the query params are ugly as a result:

![image](https://user-images.githubusercontent.com/9373513/73848751-51e56c80-47ee-11ea-9847-2a29eb5e8b7e.png)

We can resolve this in the future though, but it will have to be very near future and definitely before release otherwise we will get broken urls when we change the scheme.